webhook-relay/entry_point.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+WEBHOOK_PROXY_URL=$(op read op://$OP_VAULT_ID/Airflow\ Webhook/smee/address)
```

Comment:
example:

1PW secret ➜ environment variable

Reply:
how is this instance authenticated with onepassword? ie how is this script able to read secrets from the one password vault?